wp-content/themes/theme_name/inc/helpers/images.php

```diff
@@ -30,45 +30,26 @@ function get_post_image( $size, $post_id = null ) {
     }
 
     return [
-      'image'  => esc_url( $image_src ),
-      'width'  => intval( $image_width ),
-      'height' => intval( $image_height ),
+      'image' => $image_src,
+      'width' => $image_width,
+      'height' => $image_height
     ];
```

Comment:
We should have escaping here. Also we need to work on defining a php coding standard rules, so that I can create a `ruleset.xml` for phpcs (spaces around braces etc.)